SPIKE-NOTES.md

```diff
@@ -0,0 +1,30 @@
+## Open questions
+
+- How to test code migrated from strong-remoting and loopback (v3)? Do we want
+  to copy existing tests over? Migrate them to async/await style? Don't bother
```

Comment:
IMO the answer ideally should be definitely use async/await style, but re-write them using the tests for inspiration. If you were to reflect on how you migrated strong-remoting and loopback (v3) code over for the spike, do you feel that there were significant changes to warrant a re-write of the tests?